packages/web/src/common/store/tipping/sagas.ts

```diff
@@ -434,10 +423,9 @@ function* fetchSupportingForUserAsync({
     account?.user_id === userId
       ? account.supporting_count
       : MAX_ARTIST_HOVER_TOP_SUPPORTING + 1
-  const supportingList = yield* call(fetchSupporting, {
-    encodedUserId,
-    limit,
-    audiusBackendInstance
+  const supportingList = yield* call([apiClient, apiClient.getSupporting], {
```

Comment:
this makes sense, just so ya know ive mostly been doing `[apiClient, 'getSupporting']`, maybe a good norming thing? v minor tho

Reply:
i like the member vs the string because IDE can click into it 😇